tests/usability/test.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+rlJournalStart
+    rlPhaseStartSetup
+        rlRun "pushd data"
+    rlPhaseEnd
+
+    rlPhaseStartTest "Test command abbreviation"
+        rlRun -s "tmt run pl disc provi -h connect 2>&1" "1-255"
```

Comment:
This should work so lets check it works (exit code 0) indeed.

```suggestion
        rlRun -s "tmt run --rm pl disc provi -h local"
```

Reply:
Updated. And I also added these two lines in the following to make sure error `Unsupported discover method 'local'` is present if we use bad short name `provin`.

```bash
       rlRun -s "tmt run --rm pl disc provin -h local 2>&1" "1-255"
       rlAssertGrep "Unsupported discover method 'local'" $rlRun_LOG
```

Reply:
Hmm. isn't that new bug? The problem that `provin` isn't a valid discover option nor subcommand, why was it accepted?

Reply:
Hi @lukaszachy, should `tmt run -rm pl disc provin -h local` output like this?

```bash
$ tmt run --rm pl disc provin -h local
Usage: tmt run [OPTIONS] COMMAND1 [ARGS]... [COMMAND2 [ARGS]...]...
Try 'tmt run --help' for help.

Error: No such command 'provin'.
```

Reply:
Please open new issue, (and remove the test;) I'm not sure. 
One PoV is that 'discover' doesn't have any arguments but only options so 'provin' has to be subcommand.
So either we should fail in 'unknonw subcommand  for run' or 'unknown option of discover'